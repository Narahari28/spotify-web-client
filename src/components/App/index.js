/* Wrapper class that renders its children */

import React from 'react';

function App({ children }) {
	return (<div>{children}</div>);
}

export default App;