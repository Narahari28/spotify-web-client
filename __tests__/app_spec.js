import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/components/App';

describe('App (Snapshot)', () => {
  it('App renders properly', () => {
    const component = renderer.create(<App />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});