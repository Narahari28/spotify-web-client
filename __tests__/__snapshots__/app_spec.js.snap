// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App (Snapshot) App renders properly 1`] = `<div />`;
